ethsigner/commandline/src/test/java/tech/pegasys/ethsigner/CommandlineParserTest.java

```diff
@@ -227,9 +229,9 @@ void creatingSignerDisplaysFailureToCreateSignerText() {
     parser = new CommandlineParser(config, outputWriter, errorWriter);
     parser.registerSigner(subCommand);
 
-    final boolean result =
-        parser.parseCommandLine(
-            (validBaseCommandOptions() + subCommand.getCommandName()).split(" "));
+    List<String> cmdLine = validBaseCommandOptions();
```

Comment:
nit: final

Reply:
same again - can't as it gets reset on the next line.